### ISSUE TITLE
MultiSelect: Aria-checked attribute is not allowed

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#15246]
### Describe the bug

The aria-checked attribute should be used on elements with role=option. Move aria-checked attribute from "p-checkbox-box" to its parent "p-multiselect-item" with option role.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked

### Environment

https://primeng.org/multiselect

### Reproducer

https://primeng.org/multiselect

### Angular version

17.0.0

### PrimeNG version

17.13.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Steps to reproduce the behavior

https://primeng.org/multiselect Open multiselect overlay
